### ISSUE TITLE
Doesn't seem to work with nested model directories

### ISSUE THREAD

**Issue by jellebean** [lukeraymonddowning/poser#71]
The php artisan make:poser $modelName does not work for models in nested directories. For instance, our eloquent model's root directory is app/Models - which you _can_ configure, but they're further divided into categories like app/Models/Distribution/ShippingRoute.php and that does not seem to work?

**Comment by lukeraymonddowning** [comment]
Hi @jellebean!

Thanks for using Poser. You can point a Factory directly at a model using the `$modelName` variable on the Factory in question.

https://github.com/lukeraymonddowning/poser#models-location

So in your case:

```php
class ShippingRouteFactory extends Factory {

    protected $modelName = ShippingRoute::class;

}
```

Ignore this, I've just re-read. I'll take a look 👍

**Comment by lukeraymonddowning** [comment]
Hi @jellebean 

I've just released 3.0.0 that solves this. Thanks for bringing it to my attention!

Once upgraded, you should be able to run `php artisan make:poser Models\\Distribution\\ShippingRouteFactory` to get the corresponding factory.

**Comment by lukeraymonddowning** [comment]
Note that you'll want to use 3.0.1-beta. There was a bug where it could not locate the model. This fixes the issue by including the $modelName in the class.

Also, make sure there is a corresponding Laravel factory.
